nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -29,3 +29,25 @@ The following advisory is available for the {run-once-operator} 1.0.0:
 * This is the initial, generally available release of the {run-once-operator}. For installation information, see xref:../../../nodes/pods/run_once_duration_override/run-once-duration-override-install.adoc#run-once-duration-override-install[Installing the {run-once-operator}].
 
 // No bug fixes or known issues to list
+[id="run-once-duration-override-operator-release-notes-1-0-1"]
```

Comment:
The newest versions should be listed above the older version. so this should move up above the 1.1.0 section.

See the OSSO RNs as an example: https://docs.openshift.com/container-platform/4.13/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html

Reply:
Done!